render-wasm/src/render/shadows.rs

```diff
@@ -0,0 +1,36 @@
+use skia_safe::{self as skia, RRect};
+
+use super::RenderState;
+use crate::shapes::{Kind, Shadow};
+
+pub fn render_drop_shadow(
+    render_state: &mut RenderState,
+    shadow: &Shadow,
+    kind: &Kind,
+    path_transform: Option<&skia::Matrix>,
+) {
+    let shadow_paint = shadow.to_paint();
+    let canvas = render_state.drawing_surface.canvas();
+
+    match kind {
+        Kind::Rect(rect, None) => {
+            canvas.draw_rect(rect, &shadow_paint);
+        }
+        Kind::Rect(rect, Some(corners)) => {
+            let rrect = RRect::new_rect_radii(rect, corners);
+            canvas.draw_rrect(rrect, &shadow_paint);
+        }
+        Kind::Circle(rect) => {
+            canvas.draw_oval(rect, &shadow_paint);
+        }
+        Kind::Path(path) | Kind::Bool(_, path) => {
+            canvas.draw_path(
+                &path.to_skia_path().transform(path_transform.unwrap()),
+                &shadow_paint,
+            );
+        }
+        _ => {
+            // TODO: SVGRaw
```

Comment:
Can't SVGRaw shadows be rendered directly in the svg generated from clojure? (we aren't doing any custom stuff inside rust with the rest of fills, strokes, blurs...)